tests/components/geo_json_events/test_config_flow.py

```diff
@@ -0,0 +1,81 @@
+"""Define tests for the GeoJSON Events config flow."""
+import pytest
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.geo_json_events import DOMAIN
+from homeassistant.const import (
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+    CONF_RADIUS,
+    CONF_SCAN_INTERVAL,
+    CONF_URL,
+)
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+from tests.components.geo_json_events.conftest import URL
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+async def test_duplicate_error(
+    hass: HomeAssistant, config_entry: MockConfigEntry
+) -> None:
+    """Test that errors are shown when duplicates are added."""
+    conf = {CONF_URL: URL, CONF_LATITUDE: -41.2, CONF_LONGITUDE: 174.7, CONF_RADIUS: 25}
+    config_entry.add_to_hass(hass)
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}, data=conf
+    )
```

Comment:
Please split between `async_init` and `async_configure` to replicate real user behavior.

Reply:
OK, changed.